cmd/remote-storage/app/server.go

```diff
@@ -22,6 +22,7 @@ import (
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	_ "google.golang.org/grpc/encoding"
```

Comment:
Is this the encoding we need, or the gogo one from pkg/ ?

Reply:
earlier i added it because i was getting a warning when used the `Compression` config from `configgrpc.ClientConfig` . but now i am not getting any such warning. so removed it for now